cmd/convox/ps.go

```diff
@@ -14,8 +14,9 @@ func init() {
 	stdcli.RegisterCommand(cli.Command{
 		Name:        "ps",
 		Description: "list an app's processes",
-		Usage:       "",
 		Action:      cmdPs,
+		Usage:       "[subcommand] [args] [options]",
```

Comment:
I see

```
$ convox ps --help
convox ps: list an app's processes

Usage:
  convox ps <command> [subcommand]
```

I'm not sure what the `<command>` means here. I think it should just be `convox ps [subcommand]`